explainer.md

```diff
@@ -133,12 +133,9 @@ When a key system configuration is included in the `MediaDecodingConfiguration`,
  Here's a sample usage:
 
 ```Javascript
-let capabilitiesPromises = []
-
-// Like rMSKA(), orderedConfigs is ordered from most -> least wanted.
-for (config in orderedConfigs) {
-  capabilitiesPromises.push(navigator.mediaCapabilities.decodingInfo(config));
-}
+// Like rMSKA(), orderedMediaConfigs is ordered from most -> least wanted.
+const capabilitiesPromises = orderedMediaConfigs
```

Comment:
I think you'll want to update l.143 as well. IIUC each iteration of the loop is now an [key,value] array?

Reply:
capabilitiesPromises is still an array of Promises so that shouldn't change anything. See example below:

```js
const promises = [1, 2, 3]
    .map(i => Promise.resolve(i));

(async _ => {
  for await (const p of promises) {  
    console.log(p);
  }
})();
```
```
> 1
> 2
> 3  
```

Reply:
Ah, sorry! LGTM